decidim-surveys/lib/decidim/surveys/component.rb

```diff
@@ -50,7 +50,7 @@
   end
 
   # These actions permissions can be configured in the admin panel
-  component.actions = %w(respond)
+  component.actions = %w(response)
```

Comment:
Checking out this and other component actions, we do verbs instead of nouns. 

We should also update this at the resource permission (line 29) - and update the specs and code for sure!
```suggestion
  component.actions = %w(respond)
```